backend/app/services/adapters/task_kinds.py

```
@@ -716,15 +734,28 @@ def get_user_group_tasks_lite(
         return result, len(valid_tasks)
 
     def get_user_personal_tasks_lite(
-        self, db: Session, *, user_id: int, skip: int = 0, limit: int = 50
+        self,
+        db: Session,
+        *,
+        user_id: int,
+        skip: int = 0,
+        limit: int = 50,
+        types: List[str] = None,
```

Ruff (0.14.13)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)